packs/appserver/charts/Makefile

```diff
@@ -28,6 +28,8 @@ release: clean
 	helm init --client-only
 	helm package .
 	curl --fail -u $(CHARTMUSEUM_CREDS_USR):$(CHARTMUSEUM_CREDS_PSW) --data-binary "@$(NAME)-$(shell sed -n 's/^version: //p' Chart.yaml).tgz" $(CHART_REPO)/api/charts
+	helm repo add jenkins-x-api https://chartmuseum.build.cd.jenkins-x.io
+	helm repo update
 	rm -rf ${NAME}*.tgz%
```

Comment:
I'd have thought a `helm repo add` and `helm repo update` would need to happen before the `helm package` above?

Reply:
I only need to add the repo to helm in order to run update as the push is done with curl, update is only being used to regenerate the index-cache.

Reply:
I only need to add the repo to helm in order to run update as the push is done with curl, update is only being used to regenerate the index-cache.